src/utils/featheryClient/index.ts

```diff
@@ -417,9 +449,14 @@ export default class FeatheryClient extends IntegrationClient {
       });
   }
 
-  async submitCustom(customKeyValues: any, override = true) {
-    if (this.draft || this.noSave) return;
-    if (Object.keys(customKeyValues).length === 0) return;
+  /**
+   * Debounceable function responsible for pinging `/api/panel/custom/submit/<version>`
+   */
+  async _debouncedSubmitCustom(override: boolean) {
+    if (Object.keys(this.pendingCustomFieldUpdates).length === 0) return;
```

Comment:
shouldn't event listener attempt to be removed here as well before exit

Reply:
ah yes